source/Halibut.Tests/Support/TestAttributes/LatestAndPreviousClientAndServiceVersionsTestCasesAttribute.cs

```diff
@@ -15,18 +16,19 @@ public class LatestAndPreviousClientAndServiceVersionsTestCasesAttribute : TestC
             bool testNetworkConditions = true,
             bool testListening = true,
             bool testPolling = true,
-            bool testAsyncAndSyncClients = true
+            bool testAsyncAndSyncClients = true,
+            bool testAsyncServicesAsWell = false // False means only the sync service will be tested.
```

Comment:
Nit: Not a big thing, I don't know about the `AsWell` part. Would `testAsyncServices` be fine as is?